### PR TITLE
go sentence support integer/timestamp type to start traverse.

### PR DESCRIPTION
**Pull request by xuguruogu**
support go from integer or timestamp. Just like sentence below:

```
...| go from hash($-.id) over edge

```

### CODE REVIEW

File: src/graph/InterimResult.cpp [hunk 1]
```diff
@@ -191,11 +191,13 @@ InterimResult::buildIndex(const std::string &vidColumn) const {
         auto name = schema->getFieldName(i);
         if (vidColumn == name) {
             VLOG(1) << "col name: " << vidColumn << ", col index: " << i;
-            if (schema->getFieldType(i).type != SupportedType::VID) {
+            if (schema->getFieldType(i).type != SupportedType::INT &&
```
Comment (dangleptr) [inline src/graph/InterimResult.cpp:194]: I want to know in which situation you need to traverse from "Timestamp" data.....

Comment (xuguruogu) [reply src/graph/InterimResult.cpp:194]: For example: find out active users in the last hour/minute.

Comment (dangleptr) [reply src/graph/InterimResult.cpp:194]: > For example: find out active users in the last hour/minute.

Sorry, i did not understand it.  The time point is one vertex in your case?

Comment (xuguruogu) [reply src/graph/InterimResult.cpp:194]: yes, connect action with a timestamp(start of the minute/hour). We can find out the active actions, without index.

Comment (xuguruogu) [reply src/graph/InterimResult.cpp:194]: To my knowledge, timestamp has no difference with an integer type. I use timestamp type to be displayed more human-readable

Comment (dangleptr) [reply src/graph/InterimResult.cpp:194]: Interesting.  
The pr has no problems in logic.  But we need to discuss it futher.

Comment (dangleptr) [reply src/graph/InterimResult.cpp:194]: We both think the improvement is reasonable. Let's go on.  
Thanks for your contribution.

**Comment by dangleptr** [comment]
Please check the failed UTs

**Comment by Amber1990Zhang** [comment]
doc done

**Pull request merged**